corehq/apps/users/views/utils.py

```diff
@@ -144,7 +144,8 @@ def _get_location_case_counts(domain, location_ids):
         CaseES()
         .domain(domain)
         .owner(location_ids)
-        .aggregation(TermsAggregation('by_location', 'owner_id').size(0))
+        .aggregation(TermsAggregation('by_location', 'owner_id'))
+        .size(0)
```

Comment:
```suggestion
```
I believe this `size` call is redundant, as we set size to 0 whenever aggregations are used anyways:

https://github.com/dimagi/commcare-hq/blob/2acb74f785ca8971e7ed99a39c745931272e12ca/corehq/apps/es/es_query.py#L384-L385

Reply:
Thanks for the suggestion. This is nice 👍